tap_salesforce/__init__.py

```diff
@@ -105,28 +104,26 @@ def main_impl():
 def sync(
     sf: Salesforce,
     stream: Stream,
-    stream_id: str,
-    fields: Dict[str, Field],
-    replication_key: str,
+    table: Table,
+    fields: List[str],
     start_time: datetime,
     end_time: datetime,
     limit: Optional[int] = None,
 ):
     try:
-        for raw_record in sf.get_records(
-            stream_id,
+        for record in sf.get_records(
+            table,
             fields,
-            replication_key,
             start_time,
             end_date=end_time,
             limit=limit,
         ):
-            record = transform_record(raw_record, fields)
```

Comment:
We no longer need to transform the records before writing them?

Reply:
I thought it was stupid that we do any transformation on the raw record. I like to see the data just the way it is.